src/webgl/interaction.js

```diff
@@ -132,18 +132,24 @@ p5.prototype.orbitControl = function(
   }
   this._renderer.prevTouches = this.touches;
 
-  // flags for interaction
-  let executeZoomProcess = false;
-  let executeRotateProcess = false;
-  let executeMoveProcess = false;
+  // The idea of using damping is based on the following website. thank you.
+  // https://github.com/freshfork/p5.EasyCam/blob/master/p5.easycam.js
+
+  // A flag that determines whether to accelerate
+  let accelerateZoomVelocity = false;
+  let accelerateRotateVelocity = false;
+  let accelerateMoveVelocity = false;
   // variables for interaction
   let deltaRadius, deltaPhi, deltaTheta;
   let moveDeltaX, moveDeltaY;
-
+  // constants for dampingProcess
+  const damping = 0.85;
+  const rotateAccelerationFactor = 0.6;
+  const moveAccelerationFactor = 0.15;
   // For touches, the appropriate scale is different
   // because the distance difference is multiplied.
-  const mouseZoomScaleFactor = 0.02;
-  const touchZoomScaleFactor = 0.0008;
+  const mouseZoomScaleFactor = 0.00008;
```

Comment:
I think my impression of zooming using my laptop trackpad is that it feels just a little bit slow to start, how would you feel about increasing this a little bit?

Reply:
I made it a little bigger.
```js
  const mouseZoomScaleFactor = 0.0001;
  const touchZoomScaleFactor = 0.0004;
```
how is it? If possible, I would like you to rewrite the code and try it...
Here is the rewritten code. [newOrbitControl](https://openprocessing.org/sketch/1924658)